libraries/chain/resource_limits.cpp

```diff
@@ -212,6 +212,7 @@ void resource_limits_manager::add_pending_ram_usage( const account_name account,
    if (ram_delta == 0) {
       return;
    }
+   wlog("Adding pending RAM usage of ${ram_delta} to account ${account}", ("ram_delta", ram_delta)("account", account));
```

Comment:
If this is a warning just to make sure it gets printed out, put that in a comment
if that isn't the reason, please make the warning verbiage more explicit (or else add a comment in response to this if I'm just being daft.